### DISCUSSION TITLE
How to add more wifi-cards?

### DISCUSSION THREAD

**Discussion by netrunner-sweden** [nzymedefense/nzyme#392] [Questions]
Hi, I have 4 cards.
Adding 1 of them works great!

What is the proper way to add 3 more? I have tried coma separated and list. It dose not work.

Awesome software otherwise! 

Best regards,
H

**Comment by lennartkoopmann** [comment]
Hi!

The `802_11_monitors` configuration is an array, so you can simply add as many definitions as you wish, like this:

```
802_11_monitors: [
  {
    device: wlx00c0ca971201
    channels: [1,2,3,4,5,6]
    channel_hop_command: "sudo /sbin/iwconfig {interface} channel {channel}"
    channel_hop_interval: 1
  }

  {
    device: wlx00c0ca971202
    channels: [7,8,9,10,11]
    channel_hop_command: "sudo /sbin/iwconfig {interface} channel {channel}"
    channel_hop_interval: 1
  }
]
```

There was a similar question in #369 so I'm adding this to #370 to improve documentation around this topic.
